feature/widget/message-list/build.gradle.kts

```diff
@@ -3,9 +3,13 @@ plugins {
 }
 
 dependencies {
+    api(projects.app.core)
```

Comment:
Why make `:app:core` part of `:feature:widget:message-list`'s API surface? Ideally a feature module would never make any of its dependencies part of its public API.